src/components/AddressBook/AddressBook.tsx

```diff
@@ -88,8 +88,9 @@ export const AddressBook = styled(
         <div className="overlay-actionsbar">
           <div className="mb-2">
             <Dropdown
+              fullWidth
               dropdownButtonText={isLocal ? "Local" : name}
-              className="w-full border-grey-300 border-solid border rounded-none p-3"
```

Comment:
will it be more intuitive to pass `w-full` in className instead of catering it to a special attribute like `fullWidth`?

Reply:
i believe if i want it to be flexible, the w-full should be 1 lvl deeper so cant just pass in the className from parent.